scala_proto/scala_proto_toolchain.bzl

```diff
@@ -132,6 +103,24 @@ scala_proto_toolchain = rule(
     },
 )
 
+def scalapb_toolchain(name, opts = [], **kwargs):
+    """Setups default scala protobuf (scalapb) toolchain
+
+    Args:
+    name: A unique name for this target
+    opts: scalapb generator options like 'grpc' or 'flat_package'
```

Comment:
Indent `name:` and `opts:` four more spaces. This will prevent any future breakages if we ever turn on Stardoc.

It also needs:

```py
        kwargs: remaining arguments to `scala_proto_toolchain`
```